scripts/util/detect-changed-contracts.js

```diff
@@ -34,7 +34,7 @@ async function detectChangedContract(referenceCommit, targetCommit = "HEAD") {
       transform: (line) => {
```

Comment:
I guess the comment 3 lines above (^^) does not match the change (still mention version 0.8.9 instead of 0.8.21